include/swift/AST/DiagnosticsSema.def

```diff
@@ -3502,9 +3502,10 @@ NOTE(add_where_newline, none,
 NOTE(duplicate_where, none,
      "duplicate the 'where' on both patterns to check both patterns",())
 
-ERROR(trailing_closure_requires_parens,none,
-      "trailing closure requires parentheses for disambiguation in this"
-      " context", ())
+WARNING(trailing_closure_requires_parens,none,
+        "trailing closure in this context is confusable with the body of the"
```

Comment:
```suggestion
        "trailing closure in this context is confusable with the body of the "
```
Could you update?

Reply:
Sorry about that, must have snuck in when I wrapped the line to 80 chars! It should be fixed now